frontend/src/components/tabs/CustomTabPanel.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Typography } from '@mui/material';
+
+interface TabPanelProps extends React.HTMLAttributes<HTMLDivElement> {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+export const CustomTabPanel = (props: TabPanelProps) => {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
```

Comment:
Try to use Material UI Components instead of plain HTML

```suggestion
    <Box
```

Reply:
@SDToews Updated :)